tests/test_meta.py

```diff
@@ -34,10 +35,77 @@ def change_test_dir(request, monkeypatch):
 
 
 @skip_disabled
-def test_build():
-    assert os.system("python setup.py bdist_dumb") == 0  # pragma: no cover
+@pytest.mark.skipif(platform.system() != "Windows", reason="Currently only setup for windows")
+class TestBuild:
+    """This needs to be its own class since we need to run the setup inside an env where
+    telegram is not known"""
+
+    @classmethod
+    def setup_class(cls):
+        """Added so we can setup a venv folder"""
+        os.system("python -m venv venv_setup_test")
+
+    @classmethod
+    def teardown_class(cls):
+        """Remove the venv folder and other build stuff"""
+        if platform.system() == "Windows":
+            os.system('for %d in (build dist venv_setup_test) do rmdir "%~d" /s /q')
+
+    def test_build(self):
+        if platform.system() == "Windows":
+            assert (
+                os.system(".\\venv_setup_test\\Scripts\\python.exe setup.py bdist_dumb") == 0
+            )  # pragma: no cover
+        # what follows is a neat hack, convincing the local python venv that ptb isn't installed.
+        # There are more files (see teardown_class), but this is the one python apparently
+        # uses to discover local packages
+        if platform.system() == "Windows":
+            os.system("rmdir /s /q python_telegram_bot.egg-info")
```

Comment:
`pip uninstall` requires using `pip install .`, which is hardcoded to use _setup.py_ (I once saw this in the pip source code from a stackoverflow answer, I just can't find it right now). 

I do not get the autouse fixture comment, do you mean instead of having this setup in a class with construct and teardown function? I think this setup works nicely and self explains the code flow well.

Reply:
I see now about having to remove this.

my point about using a fixture is that it can automatically clean up the articfacts after every test withouth any need to repeat the code for that. That would be in addition to a class construct/teardown